src/discordcr/mappings/permissions.cr

```diff
@@ -0,0 +1,28 @@
+module Discord
+  enum DISCORD_PERMISSIONS
```

Comment:
This isn't actually a [flags enum](https://crystal-lang.org/docs/syntax_and_semantics/enum.html) either.